tests/config.nims

```diff
@@ -0,0 +1 @@
+switch("path", "$projectDir/../src")
```

Comment:
This is added by nimble when you create a new project with tests, and is needed for nimble to discover tests.